internal/eventstore/repository/events.go

```diff
@@ -0,0 +1,157 @@
+package repository
+
+import (
+	"context"
+	"time"
+
+	"github.com/caos/utils/logging"
+	"github.com/cockroachdb/cockroach-go/crdb"
+
+	"database/sql"
+
+	lib_models "github.com/caos/eventstore-lib/pkg/models"
+	caos_errs "github.com/caos/utils/errors"
+	"github.com/caos/utils/tracing"
+	"github.com/caos/zitadel/internal/eventstore/models"
+)
+
+const (
+	eventsTable = "eventstore.events"
+)
+
+type Event struct {
+	ID               string    `sql:"column:id;default:NULL"`
+	CreationDate     time.Time `sql:"column:creation_date;default:now()"`
+	Typ              string    `sql:"column:event_type;default:NULL"`
+	Sequence         uint64    `sql:"column:event_sequence;default:nextval('eventstore.event_seq')"`
+	PreviousSequence uint64    `sql:"column:previous_sequence;default:NULL"`
+	Data             []byte    `sql:"column:event_data;default:NULL"`
+	ModifierService  string    `sql:"column:modifier_service;default:NULL"`
+	ModifierTenant   string    `sql:"column:modifier_tenant;default:NULL"`
+	ModiferUser      string    `sql:"column:modifier_user;default:NULL"`
+	ResourceOwner    string    `sql:"column:resource_owner;default:NULL"`
+	AggregateType    string    `sql:"column:aggregate_type"`
+	AggregateID      string    `sql:"column:aggregate_id"`
+}
+
+func eventFromApp(event lib_models.Event, aggregate aggregateRoot) *Event {
```

Comment:
i think app is from app package in citadel, we should rename all this funktions in this file

Reply:
I solved the problem in a different way